File: 1541.py
x = 99999
while True:
    x = input().split()
    if x == ['0']:
        break
    a,b,c = x

    a, b, c = int(a), int(b), int(c)
    
    
    A = a * b
    t = A *100 / c
    t = (t) ** (1/2)
    print(int(t))

File: CSharp/1541.cs
using System;
using System.Linq;

class URI {

    static void Main(string[] args) 
    {
        while (true) {
            decimal[] x = Console.ReadLine().Split().Select(decimal.Parse).ToArray();
            if(x[0]==0)
            {break;}

            decimal y =x[0]*x[1];

            Console.WriteLine($"{(int)Math.Truncate(Math.Sqrt((double)(y/(x[2]/100))))}");
        }
    }

}